main.js

```diff
@@ -519,8 +519,13 @@ function generateFilename(prefix, suffix) {
     var date = new Date();
     var filename = prefix;
 
-    if (CONFIG && CONFIG.name && CONFIG.name.trim() !== '') {
-        filename = filename + '_' + CONFIG.name.trim().replace(' ', '_');
+    if (CONFIG) {
+        if (CONFIG.flightControllerIdentifier && CONFIG.flightControllerIdentifier.trim() !== '') {
```

Comment:
i don't think you need the second have of the expression, the fightcontroller identifier is not user-configurable and should never result in an empty string if it's set.  please double-check this statement and remove the .trim() check if true.